src/components/space/satellite.tsx

```diff
@@ -48,16 +50,15 @@ export const Satellite = ({ color, tle }: SatelliteProps) => {
   camera.getWorldDirection(vector);
 
   return (
-    <>
-      <mesh ref={planeteRef}>
-        <sphereGeometry args={[0.1, 8, 8]} />
-        <meshBasicMaterial color={color} />
-      </mesh>
-      <mesh ref={textRef}>
-        <Text color={color} outlineColor={0x000000} outlineWidth={0.01}>
-          {getSatelliteName(tle)}
-        </Text>
-      </mesh>
-    </>
-  );
+  <>
+  (<mesh ref={planeteRef} visible={visible}>)
```

Comment:
This line makes React Three Fiber log a warning every frame in the browser console.

```
Text is not allowed in the R3F tree! This could be stray whitespace or characters.
```

This is caused by the parentheses before and after `<mesh>`'s opening tag. Removing them fixes the issue.

```suggestion
  <mesh ref={planeteRef} visible={visible}>
```